api/v2/controllers/userController.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable padded-blocks */
+import bcrypt from 'bcrypt';
+import ENV from 'dotenv';
+import _ from 'lodash';
+import jwt from 'jsonwebtoken';
+import users from '../models/users';
+import validation from '../middleware/validation';
+
+ENV.config();
+
+class UserController {
+  // create user account
+  static async signUp(req, res) {
+    const { error } = validation.validateUserSignUp(req.body);
```

Comment:
A method should only do what it was created for, in this case, it was created to create an account and nothing else, so you should move the validation in its separate folder and apply it in the route as a middleware.